### PR TITLE
Add rerank cutoff for initial round of the search

### PR DESCRIPTION
**Pull request by Peilin-Yang**
This is esp. useful when applying query expansion on large dataset such as CluwWeb.

### CODE REVIEW

File: src/main/java/io/anserini/search/SearchArgs.java [hunk 1]
```diff
@@ -50,6 +50,11 @@
   @Option(name = "-hits", metaVar = "[number]", required = false, usage = "max number of hits to return")
   public int hits = 1000;
 
+  @Option(name = "-rerankcutoff", metaVar = "[number]", required = false, usage = "max number of hits " +
```
Comment (lintool) [inline src/main/java/io/anserini/search/SearchArgs.java]: `-rerankCutoff`?

**Comment by lintool** [comment]
My more general comment here is that, shouldn't these be reranker-specific?
We have:

```
  private int fbTerms = 20;
  private int fbDocs = 50;
  private float originalQueryWeight = 0.6f;
```
So that means we should actually have `-rm3.fbTerms`, `-rm3.fbDocs`, `-rm3.originalQueryWeight`, right?

**Comment by Peilin-Yang** [comment]
Yes, I think it makes more sense to specify RM3's options in SearchArgs.
But `rerankCutoff` is more general for any reranking model. 
So we probably can have `-rerankCutoff`, `-rm3.fbTerms`, `-rm3.originalQueryWeight`?

**Comment by lintool** [comment]
@Peilin-Yang I think you still want to change `-rerankcutoff` to `-rerankCutoff`?
A quick sanity check on a few quick regressions?

Otherwise, ready to be merged.

**Pull request merged**